src/request.ts

```diff
@@ -244,6 +244,7 @@ export async function prepareXchainRequestFulfillmentTransaction(
 		fromToken: fromTokenData,
 		toAmount: destinationTokenAmount,
 		toToken: toTokenData,
+		slippagePercentage: 0.3, // this can be low because squid will add slippage
```

Comment:
Thought: We should consider setting a maxAcceptableSlippage in `prepareXchainRequestFulfillmentTransaction`, because it's not visible there rn.

Also, if you think 0.3% is a good slippage number, then we should set that as default (and not the current 1.5%)

I added a quick commit to address this

Reply:
nice! and I agree we might want to accept it as a parameter